pinot-spi/src/main/java/org/apache/pinot/spi/config/table/QueryConfig.java

```diff
@@ -37,14 +37,27 @@
   // because by the time the server times out, the broker should already timed out and returned the response.
   private final Long _timeoutMs;
 
+  // Table config override for disable Groovy broker property
+  private final Boolean _disableGroovy;
```

Comment:
According to #7966, it looks that we would eventually want to disable Groovy by default. What's our plan on changing the default behavior?

Reply:
This PR is backward compatible. We can probably submit a separate small backward incompatible PR to simply change the default value for it